src/index.html

```diff
@@ -13,6 +13,67 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <img
+          class="card__imac"
+          src="./images/imac.jpeg"
+          alt="Imac img"
+        />
+      </div>
+
+      <section class="card__section">
+        <div class="card__info">
+          <div class="card__info card__info--model">
+            <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          </div>
+
+          <div class="card__info card__info--code">
+            <p>Product code: 195434</p>
+          </div>
+        </div>
+
+        <div class="card__stats">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__stats card__stats--reviews">
+            <span>Reviews: 5</span>
+          </div>
```

Comment:
```suggestion
            <span class="card__stats card__stats--reviews">Reviews: 5</span>
     
```
these wrappers look redundant